src/app/components/common/Pagination.tsx

```
@@ -0,0 +1,17 @@
+"use client";
+import React from "react";
+import { usePathname, useSearchParams } from "next/navigation";
+
+const Pagiantion = ({ totalPages }: { totalPages: number }) => {
+  const pathname = usePathname();
+  const searchParams = useSearchParams();
+  const currentPage = Number(searchParams.get("page")) || 1;
+  return (
+    <div>
+      <button disabled={currentPage === 1}>Previous</button>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)